shared/src/business/entities/EntityConstants.ts

```diff
@@ -548,13 +562,35 @@ export const OBJECTIONS_OPTIONS_MAP = {
 };
 export const OBJECTIONS_OPTIONS = [...Object.values(OBJECTIONS_OPTIONS_MAP)];
 
+export const INTERNAL_DOCUMENT_TYPES_REQUIRING_OBJECTION = new Set([
+  ...DOCUMENT_INTERNAL_CATEGORIES_MAP['Motion'].map(entry => {
+    return entry.documentType;
+  }),
+  'Application to Take Deposition',
+]);
+
+const EXTERNAL_DOCUMENTS_REQUIRING_OBJECTION = [
+  ...DOCUMENT_EXTERNAL_CATEGORIES_MAP['Motion'],
+  DOCUMENT_EXTERNAL_CATEGORIES_MAP['Application'].find(
+    doc => doc.eventCode === 'APLD',
+  )!,
+];
+
+export const EXTERNAL_DOCUMENT_TYPES_REQUIRING_OBJECTION = new Set(
+  EXTERNAL_DOCUMENTS_REQUIRING_OBJECTION.map(doc => doc.documentType),
+);
+
+export const EXTERNAL_DOCUMENT_CODES_REQUIRING_OBJECTION = new Set(
+  EXTERNAL_DOCUMENTS_REQUIRING_OBJECTION.map(doc => doc.eventCode),
+);
+
```

Comment:
Similar code was repeated in several places, so I extracted it.

In addition to `Application to Take Deposition`, the original code also had `Motion to Withdraw as Counsel` and `Motion to Withdraw Counsel (filed by petitioner)` (M12 and M116, respectively). Since this PR categorizes them as motions, we do not need to explicitly add them to the set.